test/deprecated/random/checkWriteThis.good

```diff
@@ -0,0 +1,7 @@
+checkWriteThis.chpl:6: In module 'Foo':
```

Comment:
This test also seems like something that should be adjusted rather than being in the deprecated section

Reply:
Also moved this test back to the library folder, but kept the deprecation warnings in the good file for now.